facturacion_mexico/config/fiscal_states_config.py

```
@@ -78,6 +91,11 @@ def is_final(cls, state):
 		"""Verifica si es un estado final."""
 		return state in cls.FINAL_STATES
 
+	@classmethod
+	def is_active(cls, state):
```

Ruff (0.15.17)

[warning] Missing return type annotation for classmethod `is_active`

(ANN206)